pages.ko/common/cli53.md

```diff
@@ -0,0 +1,36 @@
+# cli53
+
+> Amazon Route 53용 명령줄 도구.
+> 더 많은 정보: <https://github.com/barnybug/cli53>.
+
+- 도메인 나열:
+
+`cli53 list`
+
+- 도메인 생성:
+
+`cli53 create {{도메인주소.com}} --comment "{{코멘트}}"`
+
+- 바인드 존 파일을 `stdout`으로 내보내기:
+
+`cli53 export {{도메인주소.com}`
```

Comment:
@sebastiaanspeck I was looking for a comparison one by one, thank you for letting me know.

In addition, we will not force push in the future, but will immediately build a commit record for easy tracking. 

I used it often because it was convenient to force push and only one commit remained, but I will have to be careful in the future.

Reply:
@sebastiaanspeck Oh, this method doesn’t work because the default bash version of macOS is 3.

After upgrading to bash 5 version and running it, it works fine.

Thank you for telling me a good method. Thanks to you, I think I can find the error easily.